models/campaigner.rb

```diff
@@ -0,0 +1,7 @@
+class Campaigner < Sequel::Model(:campaigners)
+  include Shield::Model
```

Comment:
To clarify: the Shield::Model inclusion is only needed for the authentication model, right? I don't need to include it for all the models?